tailwind.config.js

```diff
@@ -3,14 +3,23 @@ var flattenColorPalette = require('tailwindcss/lib/util/flattenColorPalette').de
 
 module.exports = {
   theme: {
-    borderColor: {
+    borderColor: theme => ({
       color: {
         1: "var(--calcite-ui-border-1)",
         2: "var(--calcite-ui-border-2)",
         3: "var(--calcite-ui-border-3)",
         input: "var(--calcite-ui-border-input)"
-      }
-    },
+      },
+      "color-brand": theme("colors.brand"),
+      "color-brand-hover": theme("colors.brand-hover"),
+      "color-brand-press": theme("colors.brand-press"),
+      "color-info": theme("colors.info"),
+      "color-success": theme("colors.success"),
+      "color-warning": theme("colors.warning"),
+      "color-danger": theme("colors.danger"),
+      "color-danger-hover": theme("colors.danger-hover"),
+      "color-danger-press": theme("colors.danger-press"),
```

Comment:
`theme()` can only be used with "top-level theme keys, not the keys inside of each section" [[1]](https://tailwindcss.com/docs/theme#referencing-other-values)